SayTheirNames/Source/Controller/Home/Person/PersonController.swift

```diff
@@ -153,9 +160,11 @@ class PersonController: UIViewController {
         super.viewDidLoad()
         view.accessibilityIdentifier = "personView"
         self.network.fetchPersonDetails(with: person.identifier) { [weak self] result in
+            guard let self = self else { return }
             switch result {
             case .success(let response):
-                self?.configure(with: response.person)
+                self.configure(with: response.person)
+                self.hideDonationButton()
```

Comment:
Maybe we should also hide the button if we get an error response because the button is also showing in that scenario.

Reply:
Done